rticonnextdds_connector/rticonnextdds_connector.py

```diff
@@ -404,17 +423,18 @@ def __getitem__(self, field_name):
 			# This shouldn't happen
 			raise Error("Unexpected connector_binding.getAnyValueFromSamples result")
 
-	def get_dictionary(self):
+	def get_dictionary(self, member_name = None):
 		"""Gets a dictionary with the values of all the fields of this sample
 
 		The dictionary keys are the field names and the dictionary values correspond
 		to each field value. To see how nested types, sequences, and arrays are
 		represented, see :ref:`Accessing the data`.
 
-		:return: A dictionary containing all the fields of the sample.
+		:param str element_name: (Optional) The complex member within the type to obtain.
```

Comment:
The name of the parameter is member_name.

Add this too the description: The member type must be a struct, a union, a sequence or an array.